README.md

```diff
@@ -134,12 +134,13 @@ gulp.task('pre-push', guppy.src('pre-push', function (files, extra, cb) {
 }));
 ```
 
-#### guppy.stream(*hookName*)
+#### guppy.stream(*hookName, options*)
 
 > Supported hooks: `applypatch-msg`, `commit-msg`, `pre-applypatch`, `pre-commit`, 
 `prepare-commit-msg`
 
-Pass in the name of the git-hook to produce a stream of the related files.
+Pass in the name of the git-hook to produce a stream of the related files. 
+If *options* object is provided, it will be passed down to gulp.src method. 
```

Comment:
Can we reword this to something like "You can pass options as a second argument, please refer to the docs for `gulp.src` for more information on available options." and link to the gulp.src docs?